doc/source/migration.rst

```diff
@@ -1,3 +1,60 @@
 =================
 Migration process
 =================
+
+  
+1. Cone docs-exports repository 
+
+  To start the Migration Process you have to clone the Docs-Exports
+  This can be achived with the following command 
+
+  git clone https://github.com/opentelekomcloud-docs/doc-exports
+
+
+2. Run the Conversion Script
+
+  In the Doc-Export Repository is a Script which is used to convert the HTML files to the new format. 
+
+  The script can be run with the following comand:
+
+  python3 process.py xxx/umn/
+
+  Note: you have to change the file dirctory xxx/umn/ to your dirctive. 
+
+
+3. Create a new Repository
+
+
+4. Choose a License 
```

Comment:
steps 4-5 are done by Eco as part of the initial bootstraping. There is no need to list it here (or at least make this sub-items of 3.) Template repo is: https://github.com/opentelekomcloud-docs/template